samples/snapshot/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>-snapshot</artifactId>
+  <packaging>jar</packaging>
+  <name>Google Google Cloud AlloyDB Java Connector Snapshot Samples</name>
```

Comment:
Might want to grep for AlloyDB and alloydb just to get all these

Reply:
Fixed. (I removed all the samples/ files for now.)